### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by cgeglio** [cgeglio/Overlook#12]
- [x] Accessibility audits should be at 100%
- [x] A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad
- [x] ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
